docs/_blog/2025_q3_update.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "2025 Q3 Update: What Have We Been Up To?"
+description: "2025 Q3 Update on the Bitcoin Dev Kit Project"
+authors:
+    - thunderbiscuit
+date: "2025-11-06"
+tags: ["BDK", "project"]
+draft: false
+---
+
+Q3 saw a new feature release of the bdk_wallet library (`2.1.0`), with great new features being added including support for multipath descriptors, more flexible transaction building capabilities, as well as better caching when working with large wallets.
+
+Here are some of the notable releases and changes over Q3 to the software libraries we maintain:
+- **We have a new _Release Notes_ section** on the Book of BDK. Head over to the [book of BDK Release Guide section](https://bookofbdk.com/release-guide/2.2/notes/) for notes on the latest releases.
+- **The Electrum, Esplora, and Kyoto client** have all gotten new releases with some small added features and bug fixes.
+- **The Book of BDK examples have all been updated to reflect the new 2.0 APIs.** Check out [https://bookofbdk.com](https://bookofbdk.com)!
+- **The language bindings** for iOS, Android, and Python have released their latest stable version (2.2)! 🎉.
```

Comment:
`The language bindings** for iOS, Android, and Python have released their latest stable version (2.2)! 🎉.`

wondering if this should be in Q3 update since it happened just now in Q4? 

It's not a hill I'm trying to die on, and think its good to publicize, but conceptually I think about it as maybe that should be thru X/Nostr/Discord and then the blog be more of a recap of a quarter, but open to changing how I conceptualize of these blog posts.

Reply:
Good point! I got mixed up.